### PR TITLE
comment langs

### PR DESCRIPTION
**Pull request by jonnliwagg**

### CODE REVIEW

File: src/main/resources/templates/pets-edit.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
       layout:decorate="~{layout}">
 
 <head>
-    <title>Edit Pet Post</title>
+    <title>Edit Pet Post hahaaha</title>
```
Comment (troy9393) [inline src/main/resources/templates/pets-edit.html:6]: Good job, Jon!

**Review by troy9393 — APPROVED** [review]

File: src/main/resources/templates/pets-edit.html (hunk 1 above)
Comment (james171001) [reply src/main/resources/templates/pets-edit.html:6]: tama yan babagal ang codes

Comment (jonnliwagg) [reply src/main/resources/templates/pets-edit.html:6]: Thanks!

**Review by james171001 — APPROVED** [review]

**Comment by james171001** [comment]
looks good keep it up

**Pull request merged**